### ISSUE TITLE
[Feature Request] amp-carousel on desktop should show controls on hover

### ISSUE THREAD

**Issue by nainar** [ampproject/amphtml#17318]
amp-carousel next/prev icons should disappear after a timeout on desktop too. Currently only does this on mobile.
When the user hovers over the carousel you should show the controls

This is for both carousel and slides mode.

cc @spacedino @ericlindley-g @aghassemi

**Comment by nainar** [comment]
Hi folks, 

Let me know how you feel about this proposal.

**Comment by aghassemi** [comment]
@spacedino @nainar @ericlindley-g 

I like the consistency of the proposal with mobile and amp-image-slider. Few thoughts:
1- I think for both mobile and desktop, hint should come back and fade away again when component re-enters the viewport (in addition to hover). If we don't like this, we should also not do it for amp-image-slider.
2- after fading away, the arrows should still be focusable and become visible when focused to (e.g. first tap stop for carousel should be whatever if focusable in the item and then the arrows)
3- We still need to prioritize what carousel hint is going to look like. Not sure if that decision impacts this or not. 
4- `controls` attribute should still be hounered.

**Comment by ericlindley-g** [comment]
I generally like this mechanic, I'm on the fence about applying it now, because we've gotten feedback that the current fade-out mechanic on mobile feels "broken" given the timing, and don't want to take on that risk for desktop as well. I'd like to resolve what the best UX is for this fade-out is, and then decide if it should also be applied to mobile.  WDYT?

W/r/t Ali's points, I generally agree as well:

1. This seems good before the user has interacted, but risks being a bit annoying after that, when it's clear that the user knows how to engage. Would it be low-effort to disable the reminder after the user has interacted with a given element?
2-4.  +1

**Comment by aghassemi** [comment]
> Would it be low-effort to disable the reminder after the user has interacted with a given element?

yes. (https://twitter.com/iamdevloper/status/1027486050497773568)

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @nainar Do you have any updates?

**Comment by nainar** [comment]
There is work ongoing on carousel V2, this should be tracked as a part of that.

**Comment by sparhami** [comment]
For carousel v2, `amp-base-carousel`, the current behavior is to only hide the controls if the user swipes to another slide, so this issue should not come up there, unless we want to also implement the autohiding behavior.

**Comment by nainar** [comment]
This issue is fixed for all users who upgrade their `<amp-carousel>` version to 0.2. 

For more details please see documentation here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-carousel/amp-carousel.md
